src/Paramore.Brighter.Outbox.DynamoDB/DynamoDbOutbox.cs

```diff
@@ -55,7 +56,11 @@ public DynamoDbOutbox(IAmazonDynamoDB client, DynamoDbConfiguration configuratio
         {
             _configuration = configuration;
             _context = new DynamoDBContext(client);
-            _dynamoOverwriteTableConfig = new DynamoDBOperationConfig { OverrideTableName = _configuration.TableName };
+            _dynamoOverwriteTableConfig = new DynamoDBOperationConfig
+            {
+                OverrideTableName = _configuration.TableName,
+                ConsistentRead = true
```

Comment:
Setting `ConsistentRead = true` as default for non-GSI lookups to mitigate against `NullReferenceException`s we've been seeing when calling `PostAsync`